libs/beacon/src/beacon_setup_service.cpp

```diff
@@ -440,22 +449,13 @@ BeaconSetupService::State BeaconSetupService::OnWaitForShares()
   std::lock_guard<std::mutex> lock(mutex_);
   beacon_dkg_state_gauge_->set(static_cast<uint64_t>(State::WAIT_FOR_SHARES));
 
-  auto       intersection = (coefficients_received_ & shares_received_);
-  const bool is_ok        = intersection.size() >= beacon_->manager.polynomial_degree();
-
-  if (!condition_to_proceed_ && is_ok)
-  {
-    condition_to_proceed_ = true;
-    FETCH_LOG_INFO(LOGGING_NAME, "State: ", ToString(state_machine_->state()),
-                   " Ready. Seconds to spare: ", state_deadline_ - GetTime(), " of ",
-                   seconds_for_state_);
-  }
-
   if (timer_to_proceed_.HasExpired())
```

Comment:
I removed these as they resulted in inconsistency in the test runs - nodes would fail when they shouldn't when these were included. The tests appear to pass consistently when nothing is done before the timer has expired.